steps/cli/tkn.go

```diff
@@ -34,6 +35,8 @@ var _ = gauge.Step("Start and verify pipeline <pipelineName> with param <paramNa
 			pipelineRunName := tkn.StartPipeline(pipelineName, params, workspaces, store.Namespace(), "--use-param-defaults")
 			pipelines.ValidatePipelineRun(store.Clients(), pipelineRunName, "successful", "no", store.Namespace())
 		}(value)
+
+		time.Sleep(3 * time.Second)
```

Comment:
Validate pipelinerun already has waiter to wait for pipelinerun

Reply:
I know, this is just to put log line into correct order so that it's easier to debug later. Now it prints something like
```
      * Get tags of the imagestream "perl" from namespace "openshift" and store to variable "perl-tags"  ...[PASS]
2024/09/10 11:01:31 Starting pipeline s2i-perl-pipeline with param VERSION=5.26-ubi8
2024/09/10 11:01:31 Starting pipeline s2i-perl-pipeline with param VERSION=5.32-ubi8
2024/09/10 11:01:31 Starting pipeline s2i-perl-pipeline with param VERSION=5.32-ubi9
2024/09/10 11:01:32 Pipelinerun s2i-perl-pipeline-run-dxzr9 started
2024/09/10 11:01:32 Pipelinerun s2i-perl-pipeline-run-s7vf9 started
2024/09/10 11:01:32 Pipelinerun s2i-perl-pipeline-run-nkwrc started
2024/09/10 11:01:32 validating pipeline run for success state...
2024/09/10 11:01:32 validating pipeline run for success state...
2024/09/10 11:01:32 validating pipeline run for success state...
```

You can see that 3 pipeline runs were created but you have no idea to which version `s2i-perl-pipeline-run-dxzr9` belongs to - it might be any of those three, there is no guarantee. With my change, the log will look like this
```
      * Get tags of the imagestream "perl" from namespace "openshift" and store to variable "perl-tags"  ...[PASS]
2024/09/10 11:01:31 Starting pipeline s2i-perl-pipeline with param VERSION=5.26-ubi8
2024/09/10 11:01:31 Pipelinerun s2i-perl-pipeline-run-s7vf9 started
2024/09/10 11:01:31 validating pipeline run s2i-perl-pipeline-run-s7vf9 for success state...
2024/09/10 11:01:34 Starting pipeline s2i-perl-pipeline with param VERSION=5.32-ubi8
2024/09/10 11:01:34 Pipelinerun s2i-perl-pipeline-run-dxzr9 started
2024/09/10 11:01:34 validating pipeline run s2i-perl-pipeline-run-dxzr9 for success state...
2024/09/10 11:01:37 Starting pipeline s2i-perl-pipeline with param VERSION=5.32-ubi9
2024/09/10 11:01:37 Pipelinerun s2i-perl-pipeline-run-nkwrc started
2024/09/10 11:01:37 validating pipeline run s2i-perl-pipeline-run-nkwrc for success state...

```